pkg/driver/node/mounter/systemd_mounter.go

```diff
@@ -123,6 +123,10 @@ func (m *SystemdMounter) Mount(ctx context.Context, bucketName string, target st
 		env.Set(envprovider.EnvMaxAttempts, maxAttempts)
 	}
 
+	// Always remove `--endpoint-url` from args if provided
+	// This ensures PV-level endpoint configuration is always ignored
+	args.Remove(mountpoint.ArgEndpointURL)
```

Comment:
If possible it could be nice to have a warning when the flag is ignored.

Reply:
thanks for the suggestion, I added it in both in [this commit](https://github.com/scality/mountpoint-s3-csi-driver/pull/10/commits/0e0dcf1ac5fd5c2e0cb7a6ee17875de7c383df5a)